backend/src/ee/services/license/license-service.ts

```diff
@@ -98,10 +110,7 @@ export const licenseServiceFactory = ({
       if (appCfg.LICENSE_KEY) {
         const token = await licenseServerOnPremApi.refreshLicense();
         if (token) {
-          const {
-            data: { currentPlan }
-          } = await licenseServerOnPremApi.request.get<{ currentPlan: TFeatureSet }>("/api/license/v1/plan");
-          onPremFeatures = currentPlan;
+          await syncLicenseKeyOnPremFeatures();
```

Comment:
For this one here, I think it's ideal to throw the error if there's any so that the feedback is quick... like customers know right away that something's wrong with the LICENSE_KEY

I think for the cron job, it's okay to fail silently there but for the initial bootstrap, we should throw accoridngly